packages/components/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from "react";
+import { createContext } from "react";
+import { View, ViewProps } from "react-native";
+
+
+export type AccordionContextType = {
+    isFirstItem: boolean;
+    isLastItem: boolean;
+};
+
+export const AccordionContext = createContext<AccordionContextType>({
+    isFirstItem: true,
+    isLastItem: true,
+});
+
+export type AccordionProps = {
+    singular?: boolean;
+}
+
+export const Accordion = ({
+    singular = false,
+    children,
+    ...rest
+}: React.PropsWithChildren<AccordionProps & ViewProps>) => {
```

Comment:
Is React.PropsWithChildren necessary when you are adding ViewProps?

Reply:
Good point 👍 Fixed in c814210